providers/gcp/gcp.go

```diff
@@ -213,12 +218,28 @@ func (p *Provider) addProductPath(product string) {
 }
 
 func (p *Provider) addProjectParam() {
-	params := url.Values{}
-	params.Add("project", p.SDKConfig.Core.Project)
-	p.URL.RawQuery = params.Encode()
+	q := p.URL.Query()
```

Comment:
To prevent overwrite existing query parameters, fetch existing query at first step.